pkg/collector/corechecks/oracle-dbm/statements.go

```diff
@@ -194,6 +195,33 @@ type MetricsPayload struct {
 	OracleVersion string      `json:"oracle_version,omitempty"`
 }
 
+type FQTDBMetadata struct {
+	Tables   []string `json:"dd_tables"`
+	Commands []string `json:"dd_commands"`
+}
+
+type FQTDB struct {
+	Instance       string        `json:"instance"`
+	QuerySignature string        `json:"query_signature"`
+	Statement      string        `json:"statement"`
+	FQTDBMetadata  FQTDBMetadata `json:"metadata"`
+}
+
+type FQTDBOracle struct {
+	CDBName string `json:"cdb_name,omitempty"`
+}
+
+type FQTPayload struct {
+	Timestamp    float64     `json:"timestamp,omitempty"`
+	Host         string      `json:"host,omitempty"` // Host is the database hostname, not the agent hostname
+	AgentVersion string      `json:"ddagentversion,omitempty"`
+	Source       string      `json:"ddsource"`
+	Tags         []string    `json:"tags,omitempty"`
```

Comment:
This should be `ddtags` and a `string` instead of a slice of strings. I believe right now this field is just being ignored 

here is what we are expecting [in the backend](https://github.com/DataDog/dd-go/blob/prod/database-monitoring/apps/dbm-events-processor/decoder/payloads.go#L304)

Reply:
converted it to comma separated tags & changed the json field name to `ddtags`